client/my-sites/upgrades/checkout-thank-you/header.jsx

```diff
@@ -150,7 +159,8 @@ class CheckoutThankYouHeader extends React.Component {
 
 CheckoutThankYouHeader.propTypes = {
 	isDataLoaded: React.PropTypes.bool.isRequired,
-	primaryPurchase: React.PropTypes.object
+	primaryPurchase: React.PropTypes.object,
+	failedPurchases: React.PropTypes.array
```

Comment:
Hmm, yeah, overengineered a bit. It's not being used in there. Will do, and then merge.